<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>636</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="login">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QFrame" name="verticalFrame">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>70</y>
          <width>382</width>
          <height>201</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	background: orange;
}
QFrame{
	border-radius: 8px;
}
QLineEdit{
	background-color: white;}
QFrame{
	padding: 5px;
}
.QFrame{
	border: 1px solid grey;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="cancelButton">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loginButton">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="password">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>password</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>Email</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Login</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="statusLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="email">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>ceo@test.com</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>Password</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="Dashboard">
           <attribute name="title">
            <string>Dashboard</string>
           </attribute>
           <widget class="QLabel" name="tabStatusLabel">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>30</y>
              <width>211</width>
              <height>71</height>
             </rect>
            </property>
            <property name="text">
             <string>Login Status</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="Encrypt">
           <attribute name="title">
            <string>Encrypt</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="fileChooseButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>113</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Choose File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="uploadFileLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QPushButton" name="uploadButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Upload</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="uploadProgress">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="Decrypt">
           <attribute name="title">
            <string>Decrypt</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="4" column="1">
             <widget class="QLabel" name="progressPrecent">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QTreeWidget" name="fileList">
              <property name="styleSheet">
               <string notr="true">QTreeView::item {
 border: 1px solid #d9d9d9;
    
}

QTreeView::item:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #e7effd, stop: 1 #cbdaf1);
    border: 1px solid #bfcde4;
}

QTreeView::item:selected {
    border: 1px solid #567dbc;
}

QTreeView::item:selected:active{
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6ea1f1, stop: 1 #567dbc);
}

QTreeView::item:selected:!active {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6b9be8, stop: 1 #577fbf);
}
</string>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="autoExpandDelay">
               <number>1</number>
              </property>
              <property name="indentation">
               <number>0</number>
              </property>
              <property name="itemsExpandable">
               <bool>true</bool>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>300</number>
              </attribute>
              <attribute name="headerMinimumSectionSize">
               <number>21</number>
              </attribute>
              <attribute name="headerShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="headerStretchLastSection">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="usernameLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>User Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="userList">
                <property name="editable">
                 <bool>false</bool>
                </property>
                <property name="currentText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="getFileListButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Refresh file list</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="fileNameLineEdit"/>
              </item>
              <item>
               <widget class="QPushButton" name="downloadFileButton">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Download File</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="deleteFileButton">
                <property name="text">
                 <string>Delete File</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <widget class="QProgressBar" name="downloadProgress">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="downloadStatus">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="ManageUsers">
           <attribute name="title">
            <string>Manage</string>
           </attribute>
           <widget class="QWidget" name="gridLayoutWidget">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>449</width>
              <height>265</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="5" column="1">
              <widget class="QLineEdit" name="newUserPassword"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="newUserEmailLabel">
               <property name="text">
                <string>User Email</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="newUserLastName"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="newUserFNameLabel">
               <property name="text">
                <string>First Name</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="newUserSuperior"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="newUserLNameLabel">
               <property name="text">
                <string>Last Name</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="newPasswordLabel">
               <property name="text">
                <string>Enter New Password</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="superiorLabel">
               <property name="text">
                <string>Choose Superior</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="newUserFirstName"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="newUserDepartment"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="newUserDeptLabel">
               <property name="text">
                <string>Department</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="newUserEmail"/>
             </item>
             <item row="6" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="insertUserButton">
                 <property name="text">
                  <string>Insert New User</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="gridLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>310</y>
              <width>449</width>
              <height>71</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="removeUserButton">
                 <property name="text">
                  <string>Remove User</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="userToRemove"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="superiorLabel_2">
               <property name="text">
                <string>User To Remove       </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="Logout">
           <attribute name="title">
            <string>Logout</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
